ansible/README.md

```diff
@@ -156,7 +156,7 @@ data that include actions and activations are lost.
 Use `ansible-playbook -i environments/<environment> openwhisk.yml` to avoid wiping the data store. This is useful to start OpenWhisk after restarting your Operating System.
 
 ### Verification after Deployment
-After a successful deployment you can use the `wsk` CLI (located in the `bin` folder of the repository) to verify that OpenWhisk is operable.
+After a successful deployment you can use the `wsk` CLI (located in the `bin\go-cli` folder of the repository) to verify that OpenWhisk is operable.
```

Comment:
`bin/go-cli`
actually why not make the new CLI in `bin` we no longer need the `go-cli` distinction.